### PR TITLE
Improvement nxp 23488 functional tests for shibboleth

### PR DESCRIPTION
**Pull request by funshodavid**
Tests ignored until full CI jobs are deployed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,34 +9,32 @@
...
-      <artifactId>nuxeo-platform-login-shibboleth</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-usermapper</artifactId>
-    </dependency>
-  </dependencies>
+  <modules>
+    <module>marketplace</module>
+  </modules>
```
Comment (kevinleturc) [inline pom.xml]: Maybe add `ftest/webdriver` module ?

File: pom.xml [hunk 2]
```diff
@@ -9,34 +9,32 @@
...
-          </buildFiles>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <profiles>
+    <profile>
+      <id>ftest</id>
+      <modules>
+        <module>ftest/webdriver</module>
```
Comment (kevinleturc) [inline pom.xml:23]: Might be put in the default modules declaration.

**Comment by kevinleturc** [comment]
👍

File: pom.xml (hunk 2 above)
Comment (funshodavid) [reply pom.xml:23]: Will do when enabling the tests back.

**Pull request merged**